<ui version="4.0" >
 <class>LicenseChooserWidget</class>
 <widget class="QWidget" name="LicenseChooserWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QRadioButton" name="noLicenseButton" >
       <property name="text" >
        <string>No License</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="useLicenseButton" >
       <property name="text" >
        <string>License Selected Below</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="chooserWidgets" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="attributionCheckBox" >
        <property name="text" >
         <string>Require Attribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sharingCheckBox" >
        <property name="text" >
         <string>Allow Sharing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remixingCheckBox" >
        <property name="text" >
         <string>Allow Remixing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="noCommercialCheckBox" >
        <property name="text" >
         <string>Prohibit Commercial Works</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="shareAlikeCheckBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Require Others to Share-Alike</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" native="1" name="warningWidget" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="warningIconLabel" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="warningLabel" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>License:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Jurisdiction:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>License URI:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QComboBox" name="licenseCombo" />
          </item>
          <item>
           <widget class="QComboBox" name="jurisdictionComboBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="uriEdit" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>281</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
